contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -505,9 +505,12 @@ def _multi_dispatch(self, msgs):
         Returns the last partial message that was not complete yet.
         """
         for payload in msgs[:-1]:
+            raw_req = payload.decode("utf8")
+            self.log(raw_req, level="io")
```

Comment:
Hmm, doesn't that mean logging twice, since we log from the lightningd side?  And won't we be slowing things down a *lot* to do this?

I expect io logging for plugins which tallk to other things (e.g. webservices(, but they'll have to wire it up themselves...

Reply:
Regarding logging twice, we don't log it lightningd side.

Regarding the slowdown, maybe this could be made optional ?

Reply:
Just realized that it does not condition the write on the `io` level setting.. Right, this may not be a brilliant idea.